code/model/Comment.php

```diff
@@ -576,15 +576,15 @@ public function getCMSFields()
                 ->obj('Created')
                 ->scaffoldFormField($this->fieldLabel('Created'))
                 ->performReadonlyTransformation(),
-            TextField::create('Name', $this->fieldLabel('AuthorName')),
+            TextField::create('Name', $this->fieldLabel('Name')),
```

Comment:
Looks like this change is related to translations? It may have got in my accident...